.env.example

```
@@ -1,3 +1,15 @@
+# CookieKit analytics ID
 NEXT_PUBLIC_cookieKitId
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The NEXT_PUBLIC_cookieKitId key should be with a value or have an equal sign

(KeyWithoutValue)